internal/ips/request.go

```diff
@@ -86,10 +123,14 @@ func (c *Client) Request() *cobra.Command {
 	requestIPCmd.Flags().StringVarP(&metro, "metro", "m", "", "Code of the metro where the IP Reservation will be created")
 	requestIPCmd.Flags().IntVarP(&quantity, "quantity", "q", 0, "Number of IP addresses to reserve.")
 	requestIPCmd.Flags().StringSliceVar(&tags, "tags", nil, "Tag or Tags to add to the reservation, in a comma-separated list.")
+	requestIPCmd.Flags().IntVar(&cidr, "cidr", 0, "The size of the desired subnet in bits.")
+	requestIPCmd.Flags().StringVarP(&network, "network", "n", "", "The starting address for this VRF IP Reservation's subnet")
+	requestIPCmd.Flags().StringVarP(&vrfID, "vrfID", "v", "", "Specify the VRF UUID.")
```

Comment:
We should stick with the existing format for command-line flags: `--vrf-id` matches the format we use for `--project-id`.